plugins/woocommerce/src/Caching/ObjectCache.php

```diff
@@ -164,13 +164,13 @@ private function create_cache_key_prefix(): string {
 	/**
 	 * Add an object to the cache, or update an already cached object.
 	 *
-	 * @param int|string|null $id Id of the object to be cached, if null, get_object_id will be used to get it.
 	 * @param object|array    $object The object to be cached.
+	 * @param int|string|null $id Id of the object to be cached, if null, get_object_id will be used to get it.
 	 * @param int             $expiration Expiration of the cached data in seconds from the current time, or DEFAULT_EXPIRATION to use the default value.
 	 * @return bool True on success, false on error.
 	 * @throws CacheException Invalid parameter, or null id was passed and get_object_id returns null too.
 	 */
-	public function set( $id = null, $object, int $expiration = self::DEFAULT_EXPIRATION ): bool {
+	public function set( $object, $id = null, int $expiration = self::DEFAULT_EXPIRATION ): bool {
```

Comment:
Is flipping the order safe? This is a public method in a non-internal class (that has been available since at least [6.9.0](https://github.com/woocommerce/woocommerce/blob/6.9.0/plugins/woocommerce/src/Caching/ObjectCache.php#L173)).

Reply:
I also flagged a 'similar thing' elsewhere, but probably you're right. Felt it was worth pausing and asking, though.